source/shared/cpp/ObjectModel/ColumnSet.cpp

```diff
@@ -53,6 +53,33 @@ void ColumnSet::DeserializeChildren(ParseContext& context, const Json::Value& va
         AdaptiveCardSchemaKey::Columns,
         false,                                             // isRequired
         CardElementTypeToString(CardElementType::Column)); // impliedType
+
+    auto counts = 0;
+    std::shared_ptr<Column> cur = nullptr;
+
+    // If there is one column with numeric width,
+    // set it to stretch since the ratio requires at least two columns
+    // with numeric width.
+    for (auto const& column : m_columns)
+    {
+        const auto width {column->GetWidth()};
+        const auto ch = width.at(0);
+        if (column->GetPixelWidth() == 0 &&
+            (ch != 'a' && ch != 's') &&
```

Comment:
what happens in this case? (note `A` in `Auto`)

```json
{
  "type": "ColumnSet",
  "columns": [
    {
      "width": "Auto",
      "items": [
        {}
      ]
    }
  ]
}
```

Reply:
we put lower case width when we deserilize.